### ISSUE TITLE
Toggle is fully black when its off

### ISSUE THREAD

**Issue by pedrocr** [PistonDevelopers/conrod#1199]
Toggle seems to be fully black when off and so the label is only readable when it's turned on. I assume this was note intended? The color options also only apply to when it's on.

**Comment by mitchmindtree** [comment]
Agreed this is an issue! It would be nice if the "off" colour could be specified via theme defaults and styling builder methods.

**Comment by pedrocr** [comment]
Styling and theming is always nice but the default having the background and text color the same seems just wrong, no?

**Comment by mitchmindtree** [comment]
Yeah you're totally right - the default theme needs a review! I always thought it would be cool to have a `themes.rs` example which used conrod to make a `Theme` editor, where it started with the default theme. This could be a good way to ensure the default theme actually makes sense, while also providing a nice way to allow for users to easily make their own and save them to `json` or `toml` or something (this would probably lead towards making themes serializable again too).
